features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/public/locales/en.json

```diff
@@ -395,42 +410,52 @@
   "Apis.Details.NewOverview.Documents.error": "Error in fetching documents list of the API",
   "Apis.Details.NewOverview.Documents.not.created": "Not Created",
   "Apis.Details.NewOverview.Endpoints.endpoint.security": "Endpoint Security",
   "Apis.Details.NewOverview.Endpoints.endpoints": "Endpoints",
   "Apis.Details.NewOverview.Endpoints.production.endpoint": "Production Endpoint",
-  "Apis.Details.NewOverview.Endpoints.production.not.configured": "Not Configured",
+  "Apis.Details.NewOverview.Endpoints.production.not.set": "-",
   "Apis.Details.NewOverview.Endpoints.sandbox.endpoint": "Sandbox Endpoint",
-  "Apis.Details.NewOverview.Endpoints.sandbox.not.configured": "Not Configured",
+  "Apis.Details.NewOverview.Endpoints.sandbox.not.set": "-",
+  "Apis.Details.NewOverview.Endpoints.security.not.set": "-",
   "Apis.Details.NewOverview.Lifecycle.edit": "Edit",
   "Apis.Details.NewOverview.Lifecycle.lifecycle.status": "Lifecycle Status",
   "Apis.Details.NewOverview.MediationPolicies": "Mediation Policies",
   "Apis.Details.NewOverview.MediationPolicies.edit": "Edit",
+  "Apis.Details.NewOverview.MetaData.access.control": "Access Control",
+  "Apis.Details.NewOverview.MetaData.access.control.all.tooltip": "All : The API is viewable, modifiable by all the publishers and creators.",
+  "Apis.Details.NewOverview.MetaData.access.control.tooltip": "Restricted by roles : The API can be viewable and modifiable by only specific publishers and creators with the roles that you specify",
+  "Apis.Details.NewOverview.MetaData.context:": "Context:",
+  "Apis.Details.NewOverview.MetaData.created.time": "Created Time",
+  "Apis.Details.NewOverview.MetaData.createdTime.not.set": "-",
+  "Apis.Details.NewOverview.MetaData.description": "Description",
+  "Apis.Details.NewOverview.MetaData.description.not.set": "-",
+  "Apis.Details.NewOverview.MetaData.last.updated.time": "Last Updated Time",
+  "Apis.Details.NewOverview.MetaData.lastUpdatedTime.not.set": "-",
+  "Apis.Details.NewOverview.MetaData.metadata": "Metadata",
+  "Apis.Details.NewOverview.MetaData.provider": "Provider",
+  "Apis.Details.NewOverview.MetaData.securityScheme": "API Security",
+  "Apis.Details.NewOverview.MetaData.securityScheme.not.set": "-",
+  "Apis.Details.NewOverview.MetaData.securityScheme.tooltip": "OAuth2 is used as the default security schema.",
+  "Apis.Details.NewOverview.MetaData.tags": "Tags",
+  "Apis.Details.NewOverview.MetaData.tags.not.set": "-",
+  "Apis.Details.NewOverview.MetaData.transport.tooltip": "HTTP is less secure than HTTPS and makes your API vulnerable to security threats.",
+  "Apis.Details.NewOverview.MetaData.transports": "Transports",
+  "Apis.Details.NewOverview.MetaData.transports.not.set": "-",
+  "Apis.Details.NewOverview.MetaData.type.not.set": "-",
+  "Apis.Details.NewOverview.MetaData.type:": "Type:",
+  "Apis.Details.NewOverview.MetaData.version": "Version",
+  "Apis.Details.NewOverview.MetaData.visibility.store": "Visibility on Store",
```

Comment:
please change store to devportal

Reply:
For the UI labels, let's avoid using the shortened name "devportal" and use "Developer Portal"